Translation/en/RedirectedResources/assets/abdata/h/list/01/personality_voice_c16/translation.txt

```diff
@@ -1254,1138 +1254,1138 @@
+私にゴムを付けて欲しいなら、素直に言えばいいじゃない=If you want me to put the rubber on for you, you can just tell me.
+ねぇ、女にゴムを付けてもらうのって、興奮するのかしら？=Hey, does it turn you on, having the girl put one on for you?
+もう、入れられないじゃない……私にゴムを付けさせたいのね？=Geez, you can't put it in... I want you to put on a rubber, okay?
+ほら、オチンポ貸して……ゴム付けるのも、今では楽しみの一つよ=Hey, lend me that cock of yours... slipping a rubber on it is one of life's little pleasures for me now.
+もしかして、遠回しのプロポーズなの？　そんなわけないわよね？　ゴム付けちゃうわよ=I guess this is your roundabout way of proposing to me? Oh, it isn't? Then I'll put a rubber on it.
+何ごとも経験だけど、お互い後悔はしたくないでしょ？　だから、ゴムは付けちゃうわね=Everything is an experience, but let's not do anything we'll regret, alright? So I'll stick a rubber on it.
+私、遊びで子作りなんて、ありえないと思うの。だからゴムは付けるわよ？　いいわね？=I don't think we should make a baby by messing around. That's why I'll put a rubber on you, right? That okay?
+好奇心は猫をも殺すと言うでしょう？　取返しが付かなくなる前に、ゴムは付けさせてもらうわ=Curiosity killed the cat, right? I'll put a rubber on it, before we do something we can't fix.
+んぅっ、自分のペースで入れられるっていいわね……じっくり貴方を味わえるわっ=Nhuu... you can put it in at your own pace... I want you to savor me carefully!
+はぁっ、全部入ったわ……入れる前はいつも思うの、こんなの入るのかなって……=Ahaa, I got the whole thing... right before you stuck it in, I thought about how deep you would go...
+どう責めてほしい？　リクエストにこたえるわ=How you gonna tease me? I'll respond to any request.
+んふっ、私のおまんこ、気持ちいでしょう？　中の感触、じっくり堪能してね……=Nfuu, my pussy feels good, doesn't it? Please take your time enjoying it inside me...
+んふっ、じっとして、貴方の反応を見るのも楽しいものね=Nfu, it's fun watching your reactions when we stay still.
+ぁっ、またちょっと大きくなった？　それとも、出ちゃいそうで我慢してるのかしら？=Ah! Did it just get a little bigger? Are you trying to keep from cumming?
+どうしたの？　やっぱり自分からリードしたくなった？=What's the matter? Did you want me to take the lead?
+好きなだけ出しちゃっていいのよ？　早かったりしても笑わないし、そういうのって個人差もあるから=You can cum as much as you want, okay? I won't laugh if it's too soon... I respect our individual differences.
+女にされるのはプライドが許さない？　そういう考え、古いわよ=A woman's pride doesn't allow that? That's old-school thinking.
+攻守交替する？　アナタとできるなら、私はどっちでもいいわよ？=Swapping offense and defense? If that's what we're doing, then I don't mind, okay?
+ほぅら、優しく苛めてあげる……なんてね、ちょっと言ってみたかったの=Hey, I'm gonna slowly tease you... I kinda want to hear the sounds you make.
+んぁっ、私、必死過ぎねっ……でも、余裕がないんだから、仕方ないのよっ=Nhaa! I'm too desperate...! But I can't hold back, I just can't help it!
+ほらっ、これくらいが好きでしょうっ、私、わかってきたわよっ=Hey, I can tell how much you really love this!
+私、夢中で腰振っちゃってるわっ、アナタのおちんちん、硬くて太いのがいけないのよっ=My hips are shaking like crazy! Your cock is so thick and hard!
+これくらい強い方が、男子は好きでしょ？　私、頑張るわっ=You love being a rough boy with me, don't you? I'm trying my best to keep up!
+強い方が喜んでくれるものねっ、私が頑張らないとっ=It makes you happy to get rough... I'm doing my best!
+わかる？　トンって奥に当ててるの……そこが子宮よ、そこで感じるようにしたいのっ=Can you tell what you're hitting...? That's my womb, I want to feel it there!
+はぁっ、凄い、子宮下りてきちゃう……自分から動いて、子宮下ろしちゃうぅぅっ……=Ahaa, it's amazing how my womb is coming down... keep moving, my womb's coming down...!!
```

Comment:
It's doing what? The worst part is that this translation looks to be accurate...

Reply:
This is describing cowgirl and it's basically the up & down motion. In particular it's describing her moving her hips down.

Reply:
Seems fine in that context.